utils/test_wrapper_thapi_text_pretty.sh.in

```diff
@@ -41,7 +42,8 @@ fi
 
 current_output_bt_text_pretty="${folder_name}/current.bt_text_pretty"
 
-ln -f @top_builddir@/${be}/.libs/lib${be^^}Interval.so ${folder_name}/
+# hack because opencl components and libs use just 'cl'
```

Comment:
Do you mind putting this comment at the top when we define `subdir`? 

the `ln -f` is another hack  :) (to avoid valgrind issue when babeltrace try to load our weird tracer libs :) )